src/backend/utils/load/ag_load_edges.c

```diff
@@ -171,8 +174,10 @@ int create_edges_from_csv_file(char *file_path,
     cr.object_id = object_id;
 
     while ((bytes_read=fread(buf, 1, 1024, fp)) > 0) {
-        if (csv_parse(&p, buf, bytes_read, edge_field_cb, edge_row_cb, &cr) != bytes_read) {
-            ereport(ERROR, (errmsg("Error while parsing file: %s\n", csv_strerror(csv_error(&p)))));
+        if (csv_parse(&p, buf, bytes_read, edge_field_cb,
+                      edge_row_cb, &cr) != bytes_read) {
```

Comment:
Brace needs to be on its own line -

some_function_or_command(blah blah,...)
{